### PR TITLE
Biz 32 video management list

### PR DESCRIPTION
**Pull request by RomanFedytskyi**

### CODE REVIEW

File: app/templates/components/library-add-button-record.hbs [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<div class="record">
```
Comment (dmitrij-borchuk) [inline app/templates/components/library-add-button-record.hbs]: <div class="record">
  {{yield}}
  <p>Record</p>
</div>

File: app/components/library-container.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+import Ember from 'ember';
+
+export default Ember.Component.extend({
+  isShownRecord: false,
```
Comment (dmitrij-borchuk) [inline app/components/library-container.js]: Rename to "isShownRecordPopup"

File: app/styles/variables.scss [hunk 3]
```diff
@@ -0,0 +1,2 @@
+$red-buttons-color:    #E53935;
```
Comment (dmitrij-borchuk) [inline app/styles/variables.scss]: Change to "record {bla bla}"

File: app/templates/components/library-navigation.hbs [hunk 4]
```diff
@@ -0,0 +1,22 @@
+<div class="navigation">
+  <img src="/assets/video-tab-icon.png" alt="" class="tab-icon">
+  <div class="navigation-bar-container">
+    <ul class="navigation-bar">
+      <li>
```
Comment (dmitrij-borchuk) [inline app/templates/components/library-navigation.hbs:5]: Move to array in component controller

File: backend/dal/video.js [hunk 5]
```diff
@@ -28,6 +28,15 @@ module.exports = function(connection) {
         });
       });
     },
+    getAllVideos: function () {
+      return new Promise(function (resolve, reject) {
+        let request = 'SELECT * FROM `videos`;';
+
+        connection.query(request, function (err, response) {
+          (err || !response.length) ? reject(err) : resolve(response);
```
Comment (dmitrij-borchuk) [inline backend/dal/video.js]: Remove "!response.length"

**Review by dmitrij-borchuk — COMMENTED** [review]
Remove "public/assets/menu-icon.png"

File: app/styles/library.scss [hunk 6]
```diff
@@ -0,0 +1,328 @@
+@import 'variables';
+
+//LIBRARY
+.library-wrapper {
+  background-color: #f5f5f5;
+  min-height: 100%;
```
Comment (dmitrij-borchuk) [inline app/styles/library.scss]: Remove

File: app/styles/library.scss [hunk 7]
```diff
@@ -0,0 +1,328 @@
...
+      height: 204px;
+      margin-left: 10px;
+      position: relative;
+      text-align: center;
+      vertical-align: top;
+      width: 215px;
+      overflow: hidden;
+
+      .content {
+        display: block;
```
Comment (dmitrij-borchuk) [inline app/styles/library.scss]: Remove

File: app/styles/library.scss [hunk 8]
```diff
@@ -0,0 +1,328 @@
...
+      }
+
+      .text {
+        font-size: 24px;
+        margin-top: 24px;
+        opacity: 1;
+      }
+
+      .hoverd-button {
+        display: block;
```
Comment (dmitrij-borchuk) [inline app/styles/library.scss:142]: Check

File: app/styles/library.scss [hunk 9]
```diff
@@ -0,0 +1,328 @@
...
+        height: 143px;
+        width: 100%;
+      }
+
+      .text {
+        color: rgba(0, 0, 0, 0.87);
+        font-size: 18px;
+        margin: 10px 0 0 0;
+        padding: 0 10px;
+        overflow: hidden;
```
Comment (dmitrij-borchuk) [inline app/styles/library.scss]: Move to separate class

File: app/styles/library.scss [hunk 10]
```diff
@@ -0,0 +1,328 @@
...
+          margin-right: 10px;
+        }
+
+        .menu {
+          margin-right: 10px;
+        }
+      }
+    }
+
+    .ember-view {
```
Comment (dmitrij-borchuk) [inline app/styles/library.scss]: Remove or rename

**Review by dmitrij-borchuk — CHANGES_REQUESTED** [review]

File: app/components/library-container.js (hunk 2 above)
Comment (RomanFedytskyi) [reply app/components/library-container.js]: fixed

File: app/styles/variables.scss (hunk 3 above)
Comment (RomanFedytskyi) [reply app/styles/variables.scss]: fixed

File: backend/dal/video.js (hunk 5 above)
Comment (RomanFedytskyi) [reply backend/dal/video.js]: fixed

File: app/styles/library.scss (hunk 7 above)
Comment (RomanFedytskyi) [reply app/styles/library.scss]: fixed

File: app/styles/library.scss (hunk 6 above)
Comment (RomanFedytskyi) [reply app/styles/library.scss]: fixed

File: app/styles/library.scss (hunk 8 above)
Comment (RomanFedytskyi) [reply app/styles/library.scss:142]: fixed

File: app/styles/library.scss (hunk 9 above)
Comment (RomanFedytskyi) [reply app/styles/library.scss]: fixed

File: app/styles/library.scss (hunk 10 above)
Comment (RomanFedytskyi) [reply app/styles/library.scss]: fixed

File: app/templates/components/library-add-button-record.hbs (hunk 1 above)
Comment (RomanFedytskyi) [reply app/templates/components/library-add-button-record.hbs]: fixed

**Review by dmitrij-borchuk — APPROVED** [review]

**Pull request merged**